spec/app/application.rb

```diff
@@ -34,6 +34,16 @@ class Base < Sinatra::Base
         serializer: FooSerializer
       }
     end
+
+    get '/with-each-serializer/' do
+      Test.create(foo: 'bar1')
+      Test.create(foo: 'bar2')
+
+      json Test.all, {
+        root: 'units',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.